scripts/build.sh

```diff
@@ -189,6 +189,8 @@ export ADUC_PRIVATE_KEY_PASSWORD=$ADUC_KEY_DIR/priv.pass
 # Remove the conf dir before sourcing oe-init-build-env
 if [[ $CLEAN == "true" ]]; then
     rm -rf $BUILD_DIR/conf
+    # cp $ROOT_DIR/yocto/config-templates/raspberrypi4-64/local.conf $BUILD_DIR/conf
```

Comment:
these are commented-out. remove or planning to uncomment them in the future?

Reply:
Cleanup PR is in the works :)